.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
     strategy:
       matrix:
         node-version:
-        - 16.x
+        - 16
```

Comment:
This renaming is also separate. It doesn't simplify anything as the PR title says.